server/src/main/java/org/opensearch/common/util/FeatureFlags.java

```diff
@@ -128,6 +128,13 @@ public class FeatureFlags {
         Property.NodeScope
     );
 
+    /**
+     * Gates the functionality of the Netty4 gRPC server.
+     */
+    public static final String GRPC_EXPERIMENTAL = "opensearch.experimental.feature.grpc.enabled";
+    // TODO: Defaulting to true. Must resolve issue providing port permissions in Security.java from feature flag.
+    public static final Setting<Boolean> GRPC_ENABLE_SETTING = Setting.boolSetting(GRPC_EXPERIMENTAL, true, Property.NodeScope);
```

Comment:
The default should be `false` here. We've got documentation on how to enable a feature flag, including [during development](https://opensearch.org/docs/latest/install-and-configure/configuring-opensearch/experimental/#enable-for-opensearch-development).